Src/WitsmlExplorer.Frontend/components/Modals/ServerModal.tsx

```diff
@@ -94,7 +94,7 @@ const ServerModal = (props: ServerModalProps): React.ReactElement => {
       <ModalDialog
         heading={`Remove the server "${server.name}"?`}
         content={<>Removing a server will permanently remove it from the list.</>}
-        confirmColor={"secondary"}
+        confirmColor={"danger"}
```

Comment:
There are multiple other modals that used "secondary" and now should use "danger" to match.
![image](https://user-images.githubusercontent.com/66632214/201143638-41daae37-1a79-40e3-95cc-032f70e3aba2.png)

Reply:
Fixed. (I also had to change allowed colors in ConfirmDialog)